script.js

```diff
@@ -6,13 +6,120 @@ const menuIcon = document.querySelector('.menuIcon');
 const menuItem = document.querySelector('.menu-item-desktop');
 const items = document.querySelectorAll('.menuItemDesktop');
 
+const body = document.querySelector('body');
+const cardData = [
+  {
+    name: 'Project 1',
+    description:
+      'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ipsam nobis nemo vitae tempora et magni in corporis earum illum cupiditate repudiandae, amet nihil reprehenderit voluptatum tempore eligendi deserunt id adipisci.',
+    featuredimg: './assets/img/laptop.png',
+    technologies: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
+    linklive: 'http://google.com',
+    linksrc: 'http://github.com',
```

Comment:
- [ ] Lastly, I noticed that whenever I hover over the two buttons, the text is no longer visible. I have attached images below. I will suggest that you refer to the Figma design and apply the right hover interaction styles to the buttons depending on the colors :100: 
- [ ] KIndly do same for all the popup buttons :+1: 
- Your Design on hover
![image](https://user-images.githubusercontent.com/102699958/232048345-b7bea3f3-f66e-4479-9577-9784bbf4f866.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/232048624-5d87ee3a-adf7-4d39-bd42-7c7da2c21b89.png)